ESSArch_Core/ip/tests/test_models.py

```diff
@@ -128,3 +129,43 @@ def test_list_files_in_ip_container(self, mock_fid, mock_list_files, mock_open_f
         mock_open_file.return_value
         mock_fid.return_value.get_mimetype.return_value
         mock_list_files.assert_called_once_with(path)
+
+
+class StatusTest(TestCase):
+
+    def setUp(self):
+        self.datadir = normalize_path(tempfile.mkdtemp())
+        self.ip = InformationPackage.objects.create(object_path=self.datadir)
```

Comment:
`object_path` is not required for these tests, which also means that `self.datadir` and `self.addCleanup` is no longer needed